src/Gui/PythonWrapper.cpp

```diff
@@ -163,11 +167,11 @@ constexpr const char* ModuleShiboken            = "shiboken2";
 #endif
 constexpr const char* ModulePySide              = "PySide2";
 #elif defined(HAVE_SHIBOKEN6)
-PyTypeObject** SbkPySide6_QtCoreTypes           = nullptr;

```

Comment:
This will fail for older PySide6 versions. Here the HAVE_SHIBOKEN_TYPEINITSTRUCT is required, too.